src/controllers/organization.ts

```diff
@@ -228,6 +251,27 @@ export const getUnableToAutoJoinOrganizationController = async (
   }
 };
 
+export function getUnableToCertifyUserAsExecutiveController(
+  req: Request,
+  res: Response,
+  next: NextFunction,
+) {
+  try {
+    return res.render("user/unable-to-certify-user-as-executive", {
+      illustration: "error.svg",
+      oidcError: "server_error",
+      interactionId: req.session.interactionId,
+      pageTitle: "Certification impossible",
+      use_dashboard_layout: false,
+    });
+  } catch (e) {
+    if (e instanceof NotFoundError) {
+      next(new HttpErrors.NotFound());
+    }
```

Comment:
suggestion: not sure handling NotFoundError here is useful. Am I missing something?

Reply:
```suggestion
```